ChangeFeedLabSolution/ChangeFeedProcessor.cs

```diff
@@ -50,25 +51,20 @@ public static class ChangeFeedProcessor
             string eventHubNamespaceConnection = ConfigurationSettings.AppSettings["EventHubNamespaceConnection"];
 #pragma warning restore CS0618 // Type or member is obsolete

```

Comment:
We should not be demonstrating obsolete members in a sample.   Any idea what the compiler is flagging here?